src/Searchlight/DataSource.cs

```diff
@@ -663,6 +673,12 @@ private static object ParseParameter(ColumnInfo column, string valueToken, strin
 
                     // All others use the default behavior
                 }
+                else if (fieldType == typeof(DateTime))
+                {
+                    var definedDate = DefinedDateOperators(valueToken);
```

Comment:
Does this crash if the valueToken isn't a defined date operator?

Can you please add a test that validates normal behavior e.g. `hired < 2021-09-21` ?

Reply:
Does this work for capitalization?  We would need to all-caps the value token before testing it.

Reply:
It did check for case in the if statement in `DefinedDateOperators` but I missed using `ToUpper()` on it when accessing the dictionary, fixing that in the next commit